src/redux/Categories/categorySlice.js

```diff
@@ -1,14 +1,14 @@
 import { createReducer } from '@reduxjs/toolkit';
 
-export const CHECK_STATUS = 'BookStore-App/CategorIES/CHECK_STATUS';
+export const SEE_STATUS = 'BookStore-App/CategorIES/CHECK_STATUS';
 
 const initialState = {
   status: '',
 };
 
 const categoryReducer = createReducer(initialState, (construct) => {
   construct
-    .addCase(CHECK_STATUS, (state, action) => {
+    .addCase(SEE_STATUS, (state, action) => {
       const newState = { ...state, status: action.payload };
       return newState;
     });
```

Comment:
If you were using the `createSlice`, your set-up would have a reducer called checkStatus. This reducer will be exported in order to help your components consume them, Kindly add this to stay consistent with the milestone requirement. You can check out this [example](https://redux-toolkit.js.org/api/createSlice#:~:text=import%20%7B%20createSlice,counterSlice.reducer)